lib/execution_engine2/utils/KafkaUtils.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+"""Module to provide kafka handlers for internal logging facility."""
+
+import json
+import logging
+
+from execution_engine2.db.models.models import Status
+
+logging.basicConfig(level=logging.INFO)
+from dataclasses import dataclass
+
+from confluent_kafka import Producer
+
+KAFKA_EVENT_TYPE = "job_status_update"
+CONDOR_EVENT_TYPE = "condor_request"
+DEFAULT_TOPIC = "ee2"
+
+
+def send_kafka_update_status(data):
```

Comment:
Why have these five separate methods that don't seem to do anything? You could just call something like `send_message_to_kafka(msg)` where `msg` is an instance of a dataclass